platform/tron/stake.go

```diff
@@ -0,0 +1,102 @@
+package tron
+
+import (
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/logger"
+	services "github.com/trustwallet/blockatlas/services/assets"
```

Comment:
I like this idea to keep in separate file as `tron/stake.go` for each interface, makes it clean

Reply:
yes, I did start the refactor of the API, but I'm starting to be crazy with a lot of methods the `api.go` has. I did split the stake things to be more clear, we can do that for every API